Projects/1-Beginner/Dec2Bin.md

```
@@ -0,0 +1,36 @@
+## Bonus features
+
+-   [non-numeric charecters to binary] We can give option to a user to convert a group of non-numeric charecters to binary by encoding to ascii or unicode
+-   [click here for ascii information](https://en.wikipedia.org/wiki/ASCII)
+-   [click here for unicode information](https://en.wikipedia.org/wiki/Unicode)
+
+## Useful links and resources
+
+[Binary number system](https://en.wikipedia.org/wiki/Binary_number)
+
+## Example projects
+- [decimal to binary program geeks for geeks] (https://www.geeksforgeeks.org/dsa/program-decimal-binary-conversion/)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)